client/src/app/App.js

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import {Route, Routes} from 'react-router-dom';
 import {ToastContainer} from 'react-toastify';
+import {ThemeProvider} from '@mui/material/styles';
```

Comment:
we put the ThemeProvider in index.js instead of App.js, would we still need this?

Reply:
Didn't realize that themeprovider was already provided in index.js so this would be redundant